Modules/Module_Import/xASL_wrp_DCM2NII_Subject.m

```diff
@@ -43,21 +43,14 @@
     % Check subjectID
     thisSubject.subjectExport = xASL_imp_SubjectName(subjectID);
     
+	x.modules.import.imPar.visitNames = thisSubject.visitIDs;
```

Comment:
if Names is the same as IDs, why do we have these two variables with the same content? Can't we replace them by a single variable? We should have names and indices. In BIDS nomenclature this would be "values" (key = session, value = 20240218)

Reply:
Michael!!!! That kept me puzzled as well. I've tried to remove some of the useless code, but I didn't want to do larger changes to avoid extra bugs. Shall we make a new separate issue where we revisit how this whole thing works and remove things that don't make sense?

Reply:
Yes good point! (you can resolve)